docs/UserGuide.md

```diff
@@ -331,15 +617,15 @@ Format: `save`
-|      [help](#41-accessing-help-help)       | `help [COMMAND_WORD]`                                                                                                                                                                    |
-|   [option](#42-accessing-options-option)   | `option`<br>`option CONFIG_NAME`<br>`option CONFIG_NAME=NEW_VALUE`                                                                                                                       |
-|     [add](#43-adding-a-taskmodule-add)     | `add mod MODULE_CODE MODULAR_CREDITS [-d "MODULE_DESCRIPTION"]`<br>`add task "TASK_NAME" [-m MODULE_CODE] [-d "TASK_DESCRIPTION"] [-t “ESTIMATED_WORKING_TIME”]`                         |
-|    [del](#44-deleting-a-taskmodule-del)    | `del mod MODULE_CODE`<br>`del task TASK_NUMBER [-m MODULE_CODE]`                                                                                                                         |
-|   [edit](#45-editing-a-taskmodule-edit)    | <code>edit task TASK_NUMBER [-m MODULE_CODE] (-n "TASK_NAME" &#124; -d "TASK_DESCRIPTION" &#124; -t "ESTIMATED_WORKING_TIME")</code> <br> `edit mod MODULE_CODE -d "MODULE_DESCRIPTION"` |
-|      [mark](#46-marking-a-task-mark)       | <code>mark (c &#124; u) TASK_NUMBER [-m MODULE_CODE]</code>                                                                                                                              |
-|    [tag](#47-managing-custom-tags-tag)     | <code>tag (add &#124; del) [-m MODULE_CODE] TAG_NAME</code>                                                                                                                              |
-|     [list](#48-listing-all-tasks-list)     | `list [TAG_NAME]`                                                                                                                                                                        |
-| [grade](#49-setting-a-modules-grade-grade) | `grade MODULE_CODE MODULE_GRADE`                                                                                                                                                         |
-|        [gpa](#410-viewing-gpa-gpa)         | `gpa`                                                                                                                                                                                    |
-| [reset](#411-resetting-the-program-reset)  | `reset`                                                                                                                                                                                  |
-|     [save](#412-saving-your-data-save)     | `save`                                                                                                                                                                                   |
\ No newline at end of file
+|      [help](#41-accessing-help-help)       | `help [COMMAND_WORD]`<br> <br> e.g: <pre> > help <br> > help add</pre>
```

Comment:
I'm not sure if im reading this right, but does it look better if there was another column for examples?

Reply:
We don't have enough space here for the additional column. With referring to ab3, I put the example code under the format(discussed with Hee Kit already)

Reply:
For the extra subsection, I am thinking whether it would be make the section number too complex
<img width="572" alt="Screen Shot 2022-04-04 at 6 31 38 PM" src="https://user-images.githubusercontent.com/78465180/161526228-23812be7-d7ca-4d38-943b-548716fd047e.png">
For example, here the 4.1.1 would just follows 4.1, which would just follow 4.

Reply:
yup makes sense